src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+* {
+  padding: 0;
```

Comment:
it's not a good idea to reset padding with * selector. use it only for setting box-sizing property